src/requirements.txt

```diff
@@ -1,2 +1,3 @@
 obsah >= 1.4
+passlib
```

Comment:
what is passlib used for in prod?

Reply:
but that's in development, not in prod?

Reply:
Oh, you are right, I didn't notice that.